workspaces/cli-server/src/server.ts

```diff
@@ -29,9 +27,14 @@ import { Session, SessionsManager } from './sessions';
 
 const pJson = require('../package.json');
 
-const logFilePath = path.join(os.homedir(), '.optic', 'optic-daemon.log');
-fs.ensureDirSync(path.dirname(logFilePath));
-export const log = fs.createWriteStream(logFilePath);
+const opticHomePath = path.join(os.homedir(), '.optic');
+fs.ensureDirSync(path.dirname(opticHomePath));
+const stdoutLogFilePath = path.join(opticHomePath, 'optic-daemon-stdout.log');
+const stderrLogFilePath = path.join(opticHomePath, 'optic-daemon-stderr.log');
+const stdoutLog = fs.createWriteStream(stdoutLogFilePath);
+const stderrLog = fs.createWriteStream(stderrLogFilePath);
+const fileConsole = new console.Console(stdoutLog, stderrLog);
+global.console = fileConsole;
```

Comment:
Overriding a global in a shared-runtime environment for modules is border-line within a single isolated module, but a total no-go when using outside dependencies or when consumed as a dependency itself. Whatever this was solving for, it needs to be solved some other way if even remotely possible.